init.ps1

```diff
@@ -9,104 +12,36 @@ Set-StrictMode -Version 'Latest'
 }
 
-$installerPath = Join-Path -Path $PSScriptRoot -ChildPath ('.output\ProGetIntaller{0}-{1}.exe' -f $installerPath,$version)
-if( -not (Test-Path -Path $installerPath -PathType Leaf) )
+if( -not (Test-Path -Path $hubPath) )
 {
-    $uri = ('http://inedo.com/proget/download/sql/{0}' -f $version)
-    Write-Verbose -Message ('Downloading {0}' -f $uri)
-    Invoke-WebRequest -Uri $uri -OutFile $installerPath
+    Write-Error -Message 'Failed to download and extract Inedo Hub.'
 }
 
-$pgInstallInfo = Get-CProgramInstallInfo -Name 'ProGet'
-if( -not $pgInstallInfo )
-{
-    $outputRoot = Join-Path -Path $PSScriptRoot -ChildPath '.output'
-    New-Item -Path $outputRoot -ItemType 'Directory' -ErrorAction Ignore
-
-    $logRoot = Join-Path -Path $outputRoot -ChildPath 'logs'
-    New-Item -Path $logRoot -ItemType 'Directory' -ErrorAction Ignore
-
-    Write-Verbose -Message ('Running ProGet installer {0}.' -f $installerPath)
-    $logPath = Join-Path -Path $logRoot -ChildPath 'proget.install.log'
-    $installerFileName = $installerPath | Split-Path -Leaf
-    $stdOutLogPath = Join-Path -Path $logRoot -ChildPath ('{0}.stdout.log' -f $installerFileName)
-    $stdErrLogPath = Join-Path -Path $logRoot -ChildPath ('{0}.stderr.log' -f $installerFileName)
-    $argumentList = '/S','/Edition=Express',$dbParam,('"/LogFile={0}"' -f $logPath)
-    Write-Verbose ('{0} {1}' -f $installerPath,($argumentList -join ' '))
-    $process = Start-Process -FilePath $installerPath `
-                             -ArgumentList $argumentList `
-                             -Wait `
-                             -PassThru `
-                             -RedirectStandardError $stdErrLogPath `
-                             -RedirectStandardOutput $stdOutLogPath
-    $process.WaitForExit()
-
-    Write-Verbose -Message ('{0} exited with code {1}' -f $installerFileName,$process.ExitCode)
-
-    if( -not (Get-CProgramInstallInfo -Name 'ProGet') )
-    {
-        if( $runningUnderAppVeyor )
-        {
-            Get-ChildItem -Path $logRoot |
-                ForEach-Object {
-                    $_
-                    $_ | Get-Content
-                }
-        }
-        Write-Error -Message ('It looks like ProGet {0} didn''t install. The install log might have more information: {1}' -f $version, $logPath)
-    }
-}
-elseif( $pgInstallInfo.DisplayVersion -notmatch ('^{0}\b' -f [regex]::Escape($version)) )
-{
-    Write-Warning -Message ('You''ve got an old version of ProGet installed. You''re on version {0}, but we expected version {1}. Please *completely* uninstall version {0} using the Programs and Features control panel, then re-run this script.' -f $pgInstallInfo.DisplayVersion, $version)
-}
+# Free edition license
+& $hubPath 'install' `
+           "ProGet:$($version)" `
+           --ConnectionString="Server=$($SqlServerName); $($dbCredentials)" `
+           --LicenseKey=MCTT2MUA-2Y72-F16311-S89JKR-KJWRU50W

```

Comment:
Does the Inedo Hub installer require an internet connection to install the product?

Reply:
No. You can create an offline installer on a machine that has internet access then copy that package to the internet-less server.